src/app/(admin)/admin/[...not-found]/page.tsx

```diff
@@ -0,0 +1,7 @@
+import NotFoundAnimation from "components/global/NotFoundAnimation";
+
+const AdminNotFound = () => {
+  return <NotFoundAnimation href="/admin" relocatedPageName="admin home" />;
+};
```

Comment:
`const AdminNotFound = () => <NotFoundAnimation href="/admin" relocatedPageName="admin home" />`

Reply:
have you tested this on your machine? I don't think that will work...

Reply:
currently running all testing it now as you wrote that, why did you write to remove the return?

Reply:
Here's what I wrote:
`const AdminNotFound = () => <NotFoundAnimation href="/admin" relocatedPageName="admin home" />`

Here's what you wrote:
`const AdminNotFound = () => { <NotFoundAnimation href="/admin" relocatedPageName="admin home" />;};`

The first func returns a React Element whereas the second one returns nothing... the curly braces defines function logic.

Reply:
here the arrow func automatically returns the react element since theres not other func logic, but the second one doesn't since you've wrapped it in curly braces so you have to return the react element manually. since the component is doing nothing else we want to simplify it down to the first one - if you have any questions happy to chat tonight when i'm home?

Reply:
snm